massa-client/src/cmds.rs

```diff
@@ -667,3 +734,12 @@ async fn send_operation(
 pub fn parse_vec<T: std::str::FromStr>(args: &[String]) -> anyhow::Result<Vec<T>, T::Err> {
     args.iter().map(|x| x.parse::<T>()).collect()
 }
+
+fn get_file_as_byte_vec(filename: &PathBuf) -> Result<Vec<u8>> {
```

Comment:
since we are in an async context, we could use asynchronous io that wouldn't block a whole thread. Also no need to touch the metadata and risk problems if the file changes in-between or something.

```rust 
async fn get_file_as_byte_vec(filename: &PathBuf) -> Result<Vec<u8>> {
  tokio::fs::read(filename).await
}
```

Reply:
okay